.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Release
+
+on:
+  push:
+    branches: [ release ]
+
+jobs:
+
+  publish-npm:
+    runs-on: ubuntu-latest
+    environment: npm
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        registry-url: https://registry.npmjs.org/
+    - run: npm i
+    # - run: npm test
+    - name: get-version
+      id: package-version
+      uses: martinbeentjes/npm-get-version-action@v1.2.3
+    - name: Create a GitHub release
+      uses: ncipollo/release-action@v1
+      with:
+        tag: "v${{ steps.package-version.outputs.current-version}}"
+        name: "Release v${{ steps.package-version.outputs.current-version}}"
```

Comment:
@johannes-vogel @hm23 the release workflow now creates a Github release + atag on push.
Compared to our current workflow, this doesn't allow 'staging', i.e. multiple pushes to `release` before final promotion.  We can, however, defer execution of the whole workflow until someone has reviewed (see [environment rule](https://github.com/cap-js/cds-adapter-graphql/settings/environments/748766325/edit)).

Reply:
For me that's fine if we have a voter also for PRs to release branch to ensure successful tests.

Reply:
Alright.